fx_crash_sig/symbolicate.py

```diff
@@ -71,20 +72,19 @@ def __get_symbolication_request(self, stack_traces):
                 out_frame = {}
 
                 if 'ip' not in src_frame:
-                    msg = "missing ip for thread " + thread_idx + " frame "
-                    msg += frame_idx
+                    msg = "missing ip for thread " + str(thread_idx) + " frame "
+                    msg += str(frame_idx)
                     raise ValueError(msg)
 
                 ip_int = int(src_frame['ip'], 16)
                 out_frame['offset'] = src_frame['ip']
 
-                if 'module_index' not in src_frame:
+                module_index = src_frame.get('module_index')
+                if module_index is None:
                     continue
-
-                module_index = src_frame['module_index']
                 if not (module_index >= 0 and module_index < len(modules)):
-                    msg = "module_index " + module_index + " out of range for "
-                    msg += "thread " + thread_idx + " frame " + frame_idx
+                    msg = "module_index " + str(module_index) + " out of range for "
```

Comment:
We can probably assume python 3.6 and do things like:

```suggestion
                    msg = f"module_index {module_index} out of range for "
```